highlevel_tests.rb

```diff
@@ -54,6 +56,38 @@ def test_rbs_are_defined_sim_tests
     assert missing_rbs.empty?, "Error in model_tests.rb: The following Ruby tests are not in any sim_tests:\n  * #{missing_rbs.join("\n  * ")}"
   end
 
+  # Ensures that all Python files are included in an actual model_test
+  def test_pys_are_defined_sim_tests
+    all_python_paths = Dir.glob(File.join($ModelDir, '*.py'))
+    all_python_filenames = all_python_paths.map { |p| File.basename(p) }
+
+    content = File.read('model_tests.rb')
+    sim_test_re = Regexp.new('def test_.*\n(?:\s*#)*\s+result = sim_test\(\'(?<filename>.*\.py)\'\)\n(?:\s*#)*\s+end')
+    pys_in_sim_test = content.scan(sim_test_re).map(&:first)
+    missing_pys = all_python_filenames - pys_in_sim_test
+
+    expected_sim_missing = [
+      'outputcontrol_files.py'
+    ]
+    missing_pys -= expected_sim_missing
+
+    assert missing_pys.empty?, "Error in model_tests.rb: The following Python tests are not in any sim_tests:\n  * #{missing_pys.join("\n  * ")}"
+  end
+
+  def test_all_ruby_have_matching_python_tests
+    model_dir = Pathname.new($ModelDir)
+
+    expected_missing = [
+      'autosize_hvac.py'
+    ]
+
+    model_dir.glob('*.rb').each do |ruby_file|
+      python_file = ruby_file.sub_ext('.py')
+      next if expected_missing.include?(python_file.basename.to_s)
+      assert_path_exists(python_file)
+    end
+  end
+
```

Comment:
New high level tests to enforce having a matching python test for each ruby file